doc/book/app_server/cookbook.rst

```diff
@@ -774,7 +774,92 @@ An "HTML" file for this server, including Lua, could look like this
     </body>
     </html>
 
+.. _cookbook-select-all-go:
 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+select_all.go
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In Go, there is no one-liner to select all tuples from a Tarantool space.
```

Comment:
Maybe it would be better to explain that selecting all tuples is not a good idea not only in Go.

Reply:
To discuss with @artur-barsegyan.